LoginPage.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+
+const LoginSchema = Yup.object().shape({
+  email: Yup.string().email('Invalid email').required('Required'),
+  password: Yup.string().min(8, 'Too Short!').max(50, 'Too Long!').required('Required'),
+});
+
+export default function LoginPage() {
+  let history = useHistory();
+
+  return (
+    <div className="login-page">
+      <div className="image-section">
+        {/* Replace with your image */}
+        <img src="https://via.placeholder.com/150" alt="LMS" />
+      </div>
+      <div className="form-section">
+        <h1>Login</h1>
+        <p>Enter your email and password to access your account</p>
+        <Formik
+          initialValues={{ email: '', password: '' }}
+          validationSchema={LoginSchema}
+          onSubmit={(values, { setSubmitting }) => {
+            // Replace with your authentication logic
+            if (values.email === 'test@test.com' && values.password === 'password') {
+              history.push('/dashboard');
+            }
+            setSubmitting(false);
+          }}
+        >
+          {({ isSubmitting }) => (
+            <Form>
+              <Field type="email" name="email" />
```

Comment:
The authentication logic should be abstracted out to a separate function or module. This can improve the modularity and maintainability of the code. It's not a good practice to hardcode the email and password. Instead, you should use a proper authentication method.